openedx_caliper_tracking/loggers.py

```diff
@@ -8,14 +8,13 @@
 def get_formatted_log(result, filename, event_name, status_code):
     """Return a formatted log message"""
     return '{} [{}][event_type:{}] Returned status: {}'.format(
-                result, filename, event_name, status_code
-            )
+        result, filename, event_name, status_code
+    )
 
 
-def get_caliper_logger(logger_name, file_path):
+def get_test_logger(logger_name, file_path):
```

Comment:
is the name `get_test_logger` appropriate?

Reply:
The caliper test script logs using this logger. if you have any better name kindly suggest.

Reply:
Okay, that works.